modules/ROOT/pages/dataweave-cookbook-flatten-arrays.adoc

```diff
@@ -0,0 +1,316 @@
+= Flatten Elements of Arrays
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
+DataWeave can flatten subarrays of an array and collections of key-value pairs
+within DataWeave objects, arrays, and subarrays.
+//LINK TO DW 1.0 EXAMPLES:
+include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
+
+=== Flatten Subarrays with Key-Value Pairs into an Array of Objects
+
+This example shows how `flatten` acts on key-value pairs of the input array
+defined by the variable `arrayOne`. Notice that every key-value pair in the
+array becomes a separate DataWeave object.
+
+The example uses this function:
+
+* `flatten` to move the elements from the subarrays to the parent array,
```

Comment:
List format for just one item?
Suggest converting this into a plain second paragraph, or merging it into the previous.

Reply:
This is to sync with format of the other pages, which often cover multiple functions.